docs/exposing-binding-data/detect-bindings.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 6
+---
+
+# Detect Bindings through owned resources
+
+The Service Bindings operator can detect binding from resources owned by the
+service custom resource.  Service Binding operator can detect Routes, Services,
+ConfigMaps, and Secrets owned by the backing service custom resource.
+Annotation is not required to extract the binding data.
```

Comment:
Question: 

1. Does this sentence hold any value to the user concerning this topic?
2. How does this information help the user to execute this procedure? 
3. If there is no value, will removing this sentence withhold any information that blocks the user from achieving the goal mentioned in this procedure/topic?